source/style-guide/style/capitalization.txt

```diff
@@ -226,24 +224,30 @@ the terms.
    * - From the Cloud Control Panel, you can rename a check.
 
 Capitalize the Names of Product Components as Appropriate
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Follow the capitalization of major component names that's established
-by Marketing, Legal, and the product teams. However, be wary of
-overcapitalization of product terms. Not every feature or object in a
-product is a proper noun. For example, the Cloud Servers service
-enables users to create a *server*, not a *Server*. When the user
-creates a server, the user specifies an *image*, *flavor*, and
-*network*, not an *Image*, *Flavor*, and *Network*. A Performance
-server has a *data disk* and a *system disk*, not a *Data disk* and a
-*System disk*. A user uses Cloud Load Balancer to create a *load
-balancer*, not a *Load Balancer*.
-
-Many terms that might be capitalized on the interface aren't
+---------------------------------------------------------
+
+Follow the capitalization of major component names that Marketing, Legal, and the product teams have established.
+
+Be wary of overcapitalization of product terms as not every feature or object in a product is a proper noun.
+
+.. example::
+
+   - The Cloud Servers service enables users to create a *server*, not a *Server*.
+
+   - When the user creates a server, the user specifies an *image*, *flavor*, and *network*, not an *Image*, *Flavor*, and *Network*.
+
+   - A Performance server has a *data disk* and a *system disk*, not a *Data disk* and a *System disk*.
+
+   - A user uses Cloud Load Balancer to create a *load balancer*, not a *Load Balancer*.
+
+Many terms that might be capitalized in the UI aren't
 capitalized when used as common nouns. When in doubt, consult an
-existing style sheet, an editor, or the product team (but be aware that
-product teams sometimes tend to overcapitalize terms). Following are
-some tips to help you determine whether a noun should be capitalized:
+existing style sheet, an editor, or the product team.
+
+.. note:: Product teams sometimes tend to overcapitalize terms.
+
+Follow these tips to help you determine whether a noun should be
+capitalized:
```

Comment:
For me, this is all about: look it up in your current docs corpus. The hard thinking about this choice has already been done. My task, following, is merely to remain consistant.